src/algorithms/primaldual.jl

```diff
@@ -186,33 +229,45 @@ struct AFBA{R}
 end
 
 function (solver::AFBA{R})(x0::AbstractArray{C}, y0::AbstractArray{C};
-    f=Zero(), g=Zero(), h=Zero(), l=IndZero(), L=I, betaQ::R=R(0), betaR::R=R(0)
+    f=Zero(), g=Zero(), h=Zero(), l=IndZero(), L=I, beta_f::R=R(0), beta_l::R=R(0)
 ) where {R, C <: Union{R, Complex{R}}}
 
     stop(state::AFBA_state) = norm(state.FPR_x, Inf) + norm(state.FPR_y, Inf) <= solver.tol
     disp((it, state)) = @printf(
         "%6d | %7.4e\n",
-        it, norm(state.FPR_x, Inf)+norm(state.FPR_y, Inf)
+        it, norm(state.FPR_x, Inf) + norm(state.FPR_y, Inf)
     )
 
+    if (beta_f == 0 && ~( isa(f,Linear) || isa(f,Zero)) ) || (beta_l == 0 && ~is_singleton(l))
+        @warn "using Lipschiz constant zero"
+    end
+
+    if isa(h, Zero) L = R(0)*I; y0 = zero(x0) end # case h(Lx) equiv 0
```

Comment:
Indeed, it won't affect the primal variable. When relaxation parameter lambda is not equal to 1, then we end up with updating u^+ = (1-lambda) u. So that is why I set it to zero. Otherwise we can just let scale the duals throughout the iterations.